appConfigurations.js

```diff
@@ -89,6 +91,14 @@ const configureViews = app => {
   ]);
 };
 
+const client = ioRedis.createClient(
+  config.redis.host,
+  { enableOfflineQueue: false }
+);
+client.on('error', error => {
+  logger.trace(`Health check failed on redis ${error}`, 'health_check_error');
```

Comment:
do you want this as a trace or an error?

Would also be cool to push a 'event' to app insights so we can see how often redis connectivity is dropping

Reply:
This trace is for appInsight, we are using appInsight logger and trace is method of it